platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -140,7 +140,7 @@ public static synchronized Boolean isConnected() {
    *
```

Comment:
Can we add a `@deprecated` comment pointing to `LocationLayerPlugin` as in  https://github.com/mapbox/mapbox-gl-native/blob/44ca0a5ec4979138514e8dda10dd1b1173dd0db1/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/TrackingSettings.java#L24-L26

Reply:
thanks for flagging, fixed with 1d6655c